tests/tools/oecert/host/host.cpp

```diff
@@ -27,11 +27,13 @@
 #define INPUT_PARAM_OPTION_REPORT "--report"
 #define INPUT_PARAM_OPTION_EVIDENCE "--evidence"
 #define INPUT_PARAM_OPTION_ENDORSEMENTS_FILENAME "--endorsements"
+#define INPUT_PARAM_OPTION_OUT_OF_PROC "--outofproc"
```

Comment:
Consider using --out-of-proc instead.

Reply:
👍

Reply:
We need the parameter to force quote gen to be in-proc or out-of-proc (override users' setting). Something like --proc <in|out>

Reply:
Make sense.